modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaLangClassList.java

```diff
@@ -0,0 +1,261 @@
+package org.openapitools.codegen.languages;
```

Comment:
Shall we put this file under `org.openapitools.codegen` instead as `org.openapitools.codegen.languages` contains a list of generators but this class is not a generator?

Reply:
@wing328 Sorry for the late response.

I've moved the class to `org.openapitools.codegen.JavaLangClassList`.